agent/agent.go

```diff
@@ -493,10 +493,13 @@ func (agent *Agent) Handle(cmd *proto.Cmd) *proto.Reply {
 		*/
 		agent.client.Disconnect()
 		return nil // no reply
+	// @obsolete by query/plugin/os/summary
```

Comment:
It's just my comment, we don't use any tools for that (we don't even use go doc, yet)